srcd/dashboards/gitbase/welcome.json

```diff
@@ -10,7 +10,7 @@
         "created_by_fk": null,
         "json_metadata": "{\"filter_immune_slices\": [], \"timed_refresh_immune_slices\": [], \"filter_immune_slice_fields\": {}, \"expanded_slices\": {}, \"default_filters\": \"{}\", \"remote_id\": 1, \"import_time\": 1560871295, \"refresh_frequency\": 0}",
         "description": null,
-        "dashboard_title": "Welcome",
+        "dashboard_title": "Overview",

```

Comment:
it's still "welcome" few lines above: `"meta\": {\"text\": \"Welcome\"}, \"type\": \"HEADER\"}`